packages/mdc-select/_mixins.scss

```diff
@@ -176,3 +176,12 @@
     @include mdc-notched-outline-color($color);
   }
 }
+
+@mixin mdc-select-floating-label_ {
+  .mdc-floating-label {
+    @include mdc-rtl-reflexive-position(left, 16px);
+
+    bottom: 12px;
+    line-height: 1.75rem;
```

Comment:
Is this related to any of our typography variables or mixins? Or just a component-specific value?

Reply:
This is required to get the notched outline to float to the correct position. This will be refactored in a follow up PR so that the notched outline requires no changes in the default version of text field and the select.